test/ctia/entity/feed_test.clj

```diff
@@ -258,8 +258,9 @@
                                          "foouser"
                                          "foogroup"
                                          "user")
-     (let [entities (repeat 345 (assoc new-feed-maximal
```

Comment:
I agree that the speedup is probably from the entity creation, not the pagination tests themselves.

>  Couldn't one parallelize this work instead?

With only [2 cores](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources) shared by the JVM and docker (and each request involving both) I wasn't too encouraged to try parallelizing it.

Reply:
Another explanation might be that the pagination tests have to retrieve less entities. I think the first page is at least retrieved, so that's 10/2 entities, instead of 345/2 entities (based on the `total->limit` logic). Still, those numbers seem similar to a layperson.

Reply:
> With only 2 cores shared by the JVM and docker (and each request involving both) I wasn't too encouraged to try parallelizing it.

Yes I had some hope that the underlying DB workload would be a mix of io-bound and cpu-bound. May or may not be worth trying it